zenodo/modules/spam/views.py

```diff
@@ -155,7 +156,7 @@ def _evaluate_user_domain(email, normalized_emails):
     return "unclear"
 
 
-def _expand_users_info(results, include_pending=False):
+def _expand_users_info(results, include_pending=True):
```

Comment:
```suggestion
def _expand_users_info(results, include_pending=False):
```

I think by default we want to not include pending (which is anyways controlled by the querystring argument in the view)